studio/schemas/documents/videoResource.js

```diff
@@ -0,0 +1,29 @@
+export default {
+  name: 'videoResource',
+  title: 'Video Resource',
+  type: 'document',
+  fields: [
+    {
+      name: 'filename',
+      title: 'Filename',
+      type: 'string',
+      validation: (Rule) => Rule.required(),
+    },
+    {
+      name: 'originalUrl',
```

Comment:
We have this in addition to any streaming urls. We thought it would be a good idea to keep track of a video's source files.